src/lwc/geFormWidgetTokenizeCard/geFormWidgetTokenizeCard.js

```diff
@@ -42,7 +53,7 @@ export default class geFormWidgetTokenizeCard extends LightningElement {
     async connectedCallback() {
         this.PAYMENT_TRANSACTION_STATUS_ENUM = Object.freeze(JSON.parse(await getPaymentTransactionStatusValues()));
         this.domain = await getOrgDomain();
-        this.visualforceOrigin = this.buildVisualforceOriginUrl(this.domain);
+        this._visualforceOrigins = this.buildVisualforceOriginUrls(this.domain);
```

Comment:
One cleanup thing, `buildVisualforceOriginUrls` can access `this.domain` directly in the body of the function, no need to pass it as an argument.

Reply:
Done!